docs/RESUME_RESTART_GUIDE.md

```
@@ -0,0 +1,285 @@
+
+### Method 1: Command Line
+
+Simply run the same command again:
+
+```bash
+cd <project-directory>
+python <auto-claude-path>/apps/backend/run.py --spec <spec-number>
+```
+
+Example:
+```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)